src/System.Collections.NonGeneric/tests/SortedListTests.cs

```diff
@@ -995,11 +997,11 @@ public static void GetValueList_CopyTo_Invalid()
             Helpers.PerformActionOnAllSortedListWrappers(sortList1, sortList2 =>
             {
                 IList values = sortList2.GetValueList();
-                Assert.Throws<ArgumentNullException>("dest", () => values.CopyTo(null, 0)); // Array is null
-                Assert.Throws<ArgumentException>(() => values.CopyTo(new object[10, 10], 0)); // Array is multidimensional
+                AssertExtensions.Throws<ArgumentNullException>("destinationArray", "dest", () => values.CopyTo(null, 0)); // Array is null
+                Assert.Throws<ArgumentException>("array", () => values.CopyTo(new object[10, 10], 0)); // Array is multidimensional
```

Comment:
If it's null the parameter name is "destinationArray" but if it's multidimensional it's "array"?